users/views.py

```diff
@@ -79,5 +79,28 @@ def edit_profile(request):
             'p_form' : p_form}
     return render(request, 'users/edit_profile.html', args)
 
+def followUser(request):
+    if request.method == 'POST':
+        print('post')
+        print(request.POST)
+        followUserId = request.POST.get('id')
+        unfollow = request.POST.get('unfollow')
+        try:
+            user = User.objects.get(id=request.user.id)
+            followUser = User.objects.get(id=followUserId)
+            if unfollow:
+                user.profile.following.remove(followUser)
+                message = 'unfollowed {{followUser.username}}'
+            else:
+                user.profile.following.add(followUser)
+                message = 'followed {{followUser.username}}'
```

Comment:
The success message doesn't display the username but the actual string literal "unfollowed {{followUser.username}}".

Reply:
Fixed